libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/notifications/factories/NotificationCreator.kt

```diff
@@ -418,6 +418,7 @@ class DefaultNotificationCreator @Inject constructor(
                         event.imageUri?.let {
                             message.setData("image/", it)
                         }
+                        message.extras.putString(MESSAGE_EVENT_ID, event.eventId.value)
```

Comment:
Put the `eventId` into the bundle so that we may be able to find the message to redact it.